[WIP] Refactor initial commit

KahCosta opened the pull request:
Refatorando actions para utters.
/actions
domain.yml

KahCosta committed:
Refactor initial commit

KahCosta committed:
Refactor Action OqueEh
 Mudanças a serem submetidas:
	modified:   rouana/actions/actions_oqueeh.py
	modified:   rouana/domain.yml

KahCosta committed:
Refactor Actions oqueEh
 Mudanças a serem submetidas:
	modified:   rouana/actions/actions_oqueeh.py
	modified:   rouana/domain.yml

brunapinos committed:
Refactors actions "Geral"
Mudanças a serem submetidas:
	modified:   rouana/actions/actions_geral.py
	modified:   rouana/domain.yml

KahCosta committed:
Change actions OqueEh
 Mudanças a serem submetidas:
	modified:   rouana/actions/actions_oqueeh.py
	modified:   rouana/domain.yml

KahCosta committed:
Deletion "actions" file "actions_oqueeh.py"
 Mudanças a serem submetidas:
	modified:   rouana/actions/actions_oqueeh.py

brunapinos committed:
Refactors basic actions in actions_geral.py
Mudanças a serem submetidas:
	modified:   rouana/actions/actions_geral.py
	modified:   rouana/domain.yml

brunapinos committed:
Merge branch 'Refactory' of https://github.com/lappis-unb/rouana into Refactory

KahCosta committed:
Change actions in actions_curiosidades.py
 Mudanças a serem submetidas:
	modified:   rouana/actions/actions_curiosidades.py
	modified:   rouana/domain.yml

KahCosta committed:
Change actions in actions_preenchimento.py
 Mudanças a serem submetidas:
	modified:   rouana/actions/actions_preenchimento.py
	modified:   rouana/domain.yml

KahCosta committed:
Change actions in actions_erros_salic.py
 Mudanças a serem submetidas:
	modified:   rouana/actions/actions_erros_salic.py
	modified:   rouana/domain.yml

diegodorgam commented on rouana/domain.yml:
```diff
@@ -26,70 +26,677 @@ templates:
+        - - "O estado L06 acontece durante a fase de Recomposição de dano ao erário, e significa prestação de contas desaprovada com INDICATIVO para Tomada de Contas Especial"
+          - "E quando o MinC utiliza ele? Após a situação L05"
+          - "Para saber mais você pode consultar a Instrução Normativa 05/2017, art. 56"
+    utter_oqueEhSalicRouana:
```
Precisa seguir o padrão e manter o _
ex: utter_oqueEhSalic_Rouana (tais no caso)

diegodorgam commented on rouana/domain.yml:
```diff
@@ -26,70 +26,677 @@ templates:
+          - "monitoramento e prestação de contas de projetos culturais."
+    utter_oqueEhSalicProjeto:
+      - text:
+        - - "Projeto é o termo dado para a proposta aprovada no SALIC."
```
Quando tiver apenas uma linha de resposta, não precisa de 2 hífens, apenas 1.

brunapinos committed:
Changes actions' name and identations
Mudanças a serem submetidas:
	modified:   rouana/domain.yml

brunapinos committed:
Changes actions to utters and deletes actions files

KahCosta committed:
Merge branch 'master' of https://github.com/lappis-unb/rouana into Refactory

KahCosta committed:
Change actions in curiosidades.md
 Mudanças a serem submetidas:
	modified:   rouana/data/stories/curiosidades.md
	modified:   rouana/domain.yml

palomacmamede committed:
Changes in utter "cumprimentar"
Mudanças a serem submetidas:
	modified:   rouana/domain.yml

Commit:
adding slot estadoSalic

Commit:
fix nlu pipeline

Commit:
add 1st intents and stories

Commit:
change absolute paths to relative paths

Commit:
fixing domain.yml and fluxo.md

Commit:
code path adjustments

Commit:
fixed domain file - missing from ln787 ahead

palomacmamede committed:
changes in utter geralDefinirPerfil

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter geralNovaProposta

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter geralDuvidaExecucao e geralConheceProcesso

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter geralCadastroSalic

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in  utter_geralDespedir

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in geralJaEhProponente

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralIntroduzirContextoNovaProposta

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
Changes in utter_curiosidadesMais and options yes or no

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

Commit:
adding double line breaks

KahCosta committed:
Change action "oqueEh" in fluxo.md
 Mudanças a serem submetidas:
	modified:   rouana/data/stories/fluxo.md

Commit:
corrigindo intent{entity:value}

palomacmamede committed:
changes in utter_geralIntroduzirContextoNovaProposta

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes utter_default

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

KahCosta committed:
Change file curiosidades.md
 Mudanças a serem submetidas:
	modified:   rouana/data/intents/curiosidades.md

palomacmamede committed:
changes in utter_default

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
Changes in  utter_geralDuvidaExecucao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralCadastroSalic

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

KahCosta committed:
Add training in curiosidades.md
 Mudanças a serem submetidas:
	modified:   rouana/data/intents/curiosidades.md

palomacmamede committed:
changes in utter_curiosidadesIndicacao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_cumprimentar

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralDuvidaExecucao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralCadastroSalic

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralDespedir

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_oqueEhSalicLeiRouanet

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_oqueEhSalicRouana

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_oqueEhSalicMinc

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_oqueEhSalicSefic

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_oqueEhSalicAprovacao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_oqueEhSalicExecucao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_oqueEhSalicPrestacaoDeContas

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_oqueEhSalic

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_oqueEhSalicVinculada

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralExplicarDefinicaoPerfil

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralDefinirContexto

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralAviso

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralMaisAlgumaPergunta

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

KahCosta committed:
Completing training in curiosidades.md
 Mudanças a serem submetidas:
	modified:   rouana/data/intents/curiosidades.md

Commit:
ajustes de config

Commit:
Simplyfing intents

Commit:
Merge remote-tracking branch 'origin/Synonyms' into Refactory

Commit:
Inclui utterances que faltavam

Commit:
Reorganizando intents e stories

Commit:
Prrenchendo ultimas utterances

Commit:
simplify NLU pipeline

Commit:
fix duplicated intents

Commit:
fix entities classification and intents

palomacmamede committed:
utter_despedir

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_definir_perfil

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_explicar_definicao_perfil

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_nova_proposta

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in   utter_conhece_processo

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralDuvidaExecucao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralSubmissaoDeProjetos

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in  utter_geralMaisAlgumaPergunta

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_geralIntroduzirContextoNaoExecucao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_duvida_execucao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_cadastro_salic

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_ja_eh_proponente

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_curiosidades_mais_nao, utter_curiosidades_mais_sim, utter_curiosidades_mais_indicacao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_aviso

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_introduzir_contexto_nao_execucao

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in utter_mais_alguma_pergunta

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

palomacmamede committed:
changes in  utter_introduzir_contexto_nova_proposta

Signed-off-by: Paloma Mamede <palomacmamede@gmail.com>

Commit:
refinamentos

Commit:
Merge branch 'Refactory' of https://github.com/lappis-unb/rouana into Refactory

Commit:
aumentando epochs e resolvendo typos no domain.yml
